sdk/communication/azure-communication-chat/build.gradle

```diff
@@ -0,0 +1,27 @@
+ext.publishName = "Microsoft Azure Android Client Library for Communication Chat Services"
+description = "This package contains the Android client library common code for Microsoft Azure Communication Chat Service."
```

Comment:
Is it "Service" or "Services"?

Reply:
Chat 👉🏻 Service
Communication 👉🏻 Services

So "Chat Services" should become "Chat Service"

Reply:
"Services" is the correct one

Reply:
my bad. In this case, should not be plural. Changing